docs/docs/user/getting-started/setup.md

```diff
@@ -14,29 +14,29 @@ In this part, you will set up KPOps. This includes:
 
 ## Setup Kubernetes with k3d
 
-If you don't have access to an existing Kubernetes cluster, this section will guide you through creating a local cluster. We recommend the lightweight Kubernetes distribution [k3s](https://k3s.io/){target=_blank} for this. [k3d](https://k3d.io/){target=_blank} is a wrapper around k3s in Docker that lets you get started fast.
+<!-- dprint-ignore-start -->
 
-1. You can install k3d with its installation script:
+If you don't have access to an existing Kubernetes cluster, this section will guide you through creating a local cluster. We recommend the lightweight Kubernetes distribution [k3s](https://k3s.io/){target=_blank} for this. [k3d](https://k3d.io/){target=_blank} is a wrapper around k3s in Docker that lets you get started fast.
 
-   ```shell
-   wget -q -O - https://raw.githubusercontent.com/k3d-io/k3d/v5.4.6/install.sh | bash
-   ```
+   1. You can install k3d with its installation script:
```

Comment:
Do you need this indent here?
```suggestion
1. You can install k3d with its installation script:
```

Reply:
Yes, that's the fix :D

Reply:
Another fix I found is indenting nested text or items with 8 spaces instead of 8

Reply:
Fixed everything by using 4 spaces to indent instead of 3

Reply:
Turned out that I didn't need it after all.